pkg/oc/cli/admin/policy/modify_scc_test.go

```diff
@@ -120,6 +120,9 @@ func TestModifySCC(t *testing.T) {
 	}
 
 	for tcName, tc := range tests {
+		if true {
```

Comment:
Why? If you want to skip test, use `t.Skip`

Reply:
Ah, I did not realize the testing object has that method, too, thanks.

Since we're in no rush about this anymore, I'll just fix the tests instead :slightly_smiling_face: